packages/app/src/specs/InlineSpecList.cy.tsx

```diff
@@ -49,11 +49,16 @@ describe('InlineSpecList', () => {
 
     it('should support fuzzy sort', () => {
       mountInlineSpecList()
-      cy.get('input').type('scome', { force: true })
+      cy.get('input').type('compspec', { force: true })
 
       cy.get('li').should('have.length', 4)
       .should('contain', 'src/components')
       .and('contain', 'Spec-A.spec.tsx')
+
+      // Don't want the search to be too fuzzy
+      // sr => src, co => components, sp => spec
+      cy.get('input').clear().type('srcose', { force: true })
```

Comment:
Just for my knowledge - why do we have to use `force` here when typing into this input?

Reply:
Not sure, but I removed it and it's happy [06a2c94](https://github.com/cypress-io/cypress/pull/23325/commits/06a2c946f798336fdd3672c7b124c8ceaa872909)

Reply:
I suspect this is kinda-sorta my fault. The search input used to rely on absolute positioning to get the prefix & suffix elements aligned properly which resulted in DOM elements overlaying the `input`. I refactored that when adding the "X" button to clear the search input, but didn't notice these `force` calls could be removed at the same time